flow/connectors/clickhouse/normalize.go

```diff
@@ -233,7 +233,11 @@ func getOrderedOrderByColumns(
 
 	orderbyColumns := make([]string, len(orderby))
 	for idx, col := range orderby {
-		orderbyColumns[idx] = getColName(colNameMap, col.SourceName)
+		sortingKeyName := col.SourceName
+		if col.SortingKeyExpression != nil && *col.SortingKeyExpression != "" {
+			sortingKeyName = *col.SortingKeyExpression
```

Comment:
lack of validation means this opens up using `;` to run an arbitrary query. Hard to say what our threat model is, but we should explicitly get the okay that our threat model assumes anyone with access to ui has access to run arbitrary queries with any clickhouse peer